services/transaction_body.proto

```diff
@@ -375,5 +376,10 @@ message TransactionBody {
      * A transaction body for a `deleteNode` request.
      */
     NodeDeleteTransactionBody nodeDelete = 56;
+
+    /**
+     * A transaction body for a `rejectToken` request.
+     */
+    TokenRejectTransactionBody tokenReject = 57;
```

Comment:
You might need to add to `schedule_transaction_body.proto` and also for `HederaFunctionality`

Reply:
I've added it to schedule_transaction_body.proto.  🙂

For HederaFunctionality - its already [included in basic_types.proto](https://github.com/hashgraph/hedera-protobufs/pull/351/files#diff-636b14bb9e5952ee6f3253a1bba9a76bde7e8a9a27a78c0eda2d6303ebb3bde6R1219).